### ISSUE TITLE
Verbatim string

### ISSUE THREAD

**Issue by lbargaoanu** [agileobjects/ReadableExpressions#99]
A corner case with double quotes:

```C#
Expression<Func<string>> e = () => @"
12
""1""2
";
e.ToReadableString().Dump();
```
I get () => "
12
\"1\"2
"
But I think it should be smth similar to the source code, to allow for both escaped quotes and new lines.

**Comment by SteveWilkes** [comment]
Good spot! I'll sort this out.

**Comment by SteveWilkes** [comment]
This is fixed in the latest code - release to follow. Cheers!

**Comment by lbargaoanu** [comment]
Thanks!

**Comment by SteveWilkes** [comment]
This is fixed in v3.1.1 of [the NuGet package](https://www.nuget.org/packages/AgileObjects.ReadableExpressions). Cheers!
